src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
+
+.deleteConfirmationContainer {
+  display: flex;
+  align-items: center;
+  justify-content: var(--container-justify-content, flex-end);
+  column-gap: var(--container-column-gap, 30px);
```

Comment:
Here and below: `--container-column-gap` is a poor name for a CSS variable, because it isn't precise enough. Which "container"? CSS variables are global; the overrides — if they come — will come from the parent. Would you expect the parent to say `--container-column-gap` and mean "the distance between the delete button and what's to the left and to the right of it"? I wouldn't.

Reply:
i am making the gap fixed (not customisable)

Reply:
✔